.github/workflows/pr_check.yml

```diff
@@ -0,0 +1,35 @@
+name: PR Workflow
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - staging
+      - main
```

Comment:
Is it okay not to add the develop branch here?

Reply:
https://www.notion.so/uskayyyyy/Branching-Rules-0f2c7aa5e94e43d589ec1cb59f52723b?pvs=4#7eed5c2b61d84f0481c18804aa44a2f6

I assume we've agreed that for frontend, we don't have a develop branch 👀 
Is there any update about this before?

Reply:
Oh, I misunderstood.